app/assets/javascripts/discourse/app/components/glimmer-topic-timeline.hbs

```diff
@@ -1,5 +1,5 @@
 <div
-  class={{concat "timeline-container " this.classes}}
+  class={{concat "timeline-container " this.addFullscreenClass}}
```

Comment:
The name of this getter `addFullscreenClass`(?) kinda feels like it should return a boolean. Also, we have a helper for concatenating classes without needing to add random spaces to string literals. How about removing the getter and doing something like:

```suggestion
  class={{concat-class "timeline-container" (if @fullscreen "timeline-fullscreen")}}
```

Reply:
🔥